frontend/packages/kubevirt-plugin/src/components/create-vm/create-vm.tsx

```diff
@@ -225,6 +225,25 @@ export const CreateVM: React.FC<RouteComponentProps> = ({ location }) => {
     }
   }, [loaded, initData, templates, userTemplates, selectedTemplate, t]);
 
+  React.useMemo(() => {
+    const temp = selectedTemplate?.variants?.[0];
+    const requestedPvcSize =
+      temp?.objects?.[0]?.spec?.dataVolumeTemplates[0]?.spec?.pvc?.resources?.requests?.storage;
```

Comment:
@rawagner do we have selectors for this data? couldn't find.

Reply:
do we have a way to get the "rootdisk" and not just "[0]"
@metalice do you have code to find the "rootdisk" in case the first one is some other disk ?

Reply:
best way is to use VMWrapper.getBootDevice() and check if it is a 'disk' type.

Reply:
To get the VM from `selectedTemplate`, use `selectVM` selector